docs/samples/shards/UI/Layout/1.edn

```diff
@@ -0,0 +1,31 @@
+(GFX.MainWindow
+ :Contents
+ (->
+  (Setup
+   (GFX.DrawQueue) >= .ui-draw-queue
+   (GFX.UIPass .ui-draw-queue) >> .render-steps
+   (UI.LayoutClass
+    :Layout {:main_direction LayoutDirection.LeftToRight})
```

Comment:
These seem to be broken currently (still using old Layout param), can you either remove them (leave just the shs tests or fix them)

Reply:
The old .edn files have been removed.

Separately, I've updated the `run-sample.shs` to use FS.Read and it seems to be reading the sample file now, and doesn't complain that if the file is missing either, but doesn't seem to be executing the file's shards code itself, do you know how to run the code in a file @guusw?